chromium_src/chrome/browser/ui/views/frame/browser_view.h

```diff
@@ -50,6 +52,8 @@
 #undef GetTabSearchBubbleHost
 #undef BrowserViewLayout
 #undef UpdateDevToolsForContents
+#undef FullscreenStateChanged
```

Comment:
Ideally, all these should be in the opposite order of their definitions. Would you mind fixing while you are in here?

Reply:
Done